src/grpc/mod.rs

```diff
@@ -122,6 +126,67 @@ impl KeyBrokerService for KeyBroker {
         };
         Result::Ok(Response::new(reply))
     }
+
+    async fn get_online_secret(
+        &self,
+        request: Request<OnlineSecretRequest>,
+    ) -> Result<Response<OnlineSecretResponse>, Status> {
+        let r = request.into_inner();
+        let client_id = Uuid::parse_str(&r.client_id)
+            .map_err(|e| Status::internal(format!("Malformed Launch ID: {}", e)))?;
```

Comment:
We use two terms here: `client_id` and `Launch ID`. Can we choose one and stick to it?

Reply:
OK let's keep the two terms (just use the correct one in each spot).

Does the generic KBS/attestation-server have a similar concept (client_id)? What is it called there?

Reply:
They use a more sophisticated token thing for later requests.